README.md

```diff
@@ -170,6 +170,12 @@ e.g `make image-build USE_LOCAL=true"`
 
   The default image used is `quay.io/opendatahub/opendatahub-operator:dev-0.0.1` when not supply argument for `make image`
 
+- To build multi-arch image, by config env variablel PLATFORM
```

Comment:
typo.
Will wording "to build  ... do ..." sound better?

Reply:
`To build multi-arch image, config env variable PLATFORM`
sounds good?

Reply:
```suggestion
To build multi-arch image set environment variable PLATFORM
```

was in my mind, but we can check with native speakers since I'm far from it :) @grdryn

Reply:
Suggestion looks good, although maybe keeping the comma would still be good:

```suggestion
- To build multi-arch image, set environment variable PLATFORM
```

Reply:
updated